source/ShipyardPanel.cpp

```diff
@@ -101,6 +101,18 @@ void ShipyardPanel::DrawItem(const string &name, const Point &point)
 		return;
 
 	DrawShip(*ship, point, ship == selectedShip);
+
+	// Don't show the "in stock" amount if the ship has an unlimited stock.
```

Comment:
I'm confused by this comment lol
shouldn't it be
```suggestion
	// Don't show the "in stock" amount if the ship is sold.
```
BUT then again, shouldn't it still display the stock that's at reduced price + somehow denote there's more that's sold there?
I suppose you took a look at how it works for outfits? I think we gotta match that